e3-redesign/src/styles/styles.scss

```
@@ -0,0 +1,541 @@
+@import 'vendor/vendor',
+'mixins/mixins',
+'utilities/utilities',
+'vars',
+'base',
+'layout/layout',
+'elements/elements';
+
+// JS action classes
+[data-visible="false"] {
+    display: none;
+}
+
+.site-header {
+
+    @media screen and (min-width: $display-nav) {
+        padding-bottom: 20px;
+        padding-top: 20px;
+        text-align: center;
+    }
+
+    display: flow-root;
+    padding: 15px;
+
+
+    &__logo {
+        @media screen and (min-width: $display-nav) {
+            display: inline-block;
+            height: 60px;
+        }
+    }
+}
+
+.site-nav {
+
+    &__trigger {
+        position: absolute;
+        right: 0;
+        top: 0;
+
+        &[aria-expanded="true"] {
+            .menu-icon__line {
+                $yValue: 10px;
+                $deg: 45deg;
+
+                &:nth-child(1) {
+                    transform: translateY($yValue) rotate($deg);
+                }
+
+                &:nth-child(2) {
+                    opacity: 0;
+                }
+
+                &:nth-child(3) {
+                    transform: translateY(-$yValue) rotate(-$deg);
+                }
+            }
+        }
+
+        @media screen and (min-width: $display-nav) {
+            display: none;
+        }
+    }
+
+    &__list {
+
+        @media screen and (max-width: ($display-nav - 1px)) {
+            background-color: $black;
+            flex-direction: column;
+            height: 0;
+            left: 0;
+            position: absolute;
+            right: 0;
+            top: $top-height;
+            width: 100vw;
+            z-index: 4;
+
+            &[data-visible="true"] {
+                animation: expand 0.3s ease-in-out forwards; // see the expand animation below
+                // this changes the height and opacity of the mobile menu
+                display: flex;
+            }
+        }
+
+        @media screen and (min-width: $display-nav) {
+            align-content: center;
+            align-items: center;
+            display: flex;
+            flex-direction: row;
+            flex-wrap: nowrap;
+            justify-content: center;
+        }
+    }
+
+    &__list-item {
+        align-items: center;
+        display: flex;
+
+
+        @media screen and (max-width: ($display-nav - 1px)) {
+            border-bottom: 1px solid white;
+            font-size: 8px;
+            font-weight: 200;
+            justify-content: left;
+        }
+    }
+
+    &__link {
+        color: black;
+        font-size: 0.85em;
+        opacity: 1;
+        padding-bottom: 15px;
+        padding-left: 2vw;
+        padding-right: 2vw;
+        padding-top: 15px;
+
+        &:hover {
+            opacity: 0.7;
+        }
+
+        @media screen and (max-width: ($display-nav - 1px)) {
+            color: white;
+            font-size: 200%;
```

As is, the links in your mobile nav do not stretch the full width. The text itself is the link, but users are likely to have the expectation that they look like buttons and, therefore, the full block is a link. This would make that so:

```suggestion
            flex: 1 0 auto;
            font-size: 200%;
```